app/web/src/components/Workspace/WorkspaceAuditLog.vue

```diff
@@ -106,11 +104,13 @@
             <tr
               :class="
                 clsx(
-                  'h-md text-sm',
-                  themeClasses(
-                    'odd:bg-neutral-200 even:bg-neutral-100',
-                    'odd:bg-neutral-700 even:bg-neutral-800',
-                  ),
+                  'h-lg text-sm',
+                  rowCollapseState[Number(row.id)]
+                    ? 'bg-action-300'
+                    : themeClasses(
+                        'odd:bg-neutral-200 even:bg-neutral-100 hover:bg-action-200',
+                        'odd:bg-neutral-700 even:bg-neutral-800 hover:bg-action-200',
+                      ),
```

Comment:
We'll nail these colors and style in a follow-up PR, but this was aimed at just making it a little better in the meantime.